src/osd/OSD.h

```diff
@@ -1934,15 +1934,15 @@ class OSD : public Dispatcher,
 	}
       }
     }
-    bool _enqueue(PG *pg) {
+    bool _enqueue(PG *pg) override {
```

Comment:
@tchaikov 
I don't have much experience with noexcept, so please correct me if i'm wrong. 
My understanding is that we use noexcept for functions that are guaranteed not to throw exceptions, but here the push_back might throw bad_alloc exception?